webproject/src/main/java/com/google/sps/services/implementations/FetchMyOrdersServiceImpl.java

```diff
@@ -34,8 +34,10 @@ public FetchOrdersResponse execute(FetchOrdersRequest fetchOrdersRequest) throws
         List<Order> orders = orderManager.getMyOrders(roomId, userEmail);
         FetchOrdersResponse.Builder fetchOrdersResponseBuilder = FetchOrdersResponse.newBuilder();
 
-        for (Order order : orders) {
-            fetchOrdersResponseBuilder.addOrders(order);
+        if (orders != null) {
```

Comment:
why order can be null? it should return empty list instead of null, and throw exception if theres error

Reply:
Understood! Will change it

Reply:
It returns null because currently the implementation is that if there are no orders in MyOrder, it returns an empty Optional. I will change it such that it returns an empty list instead